deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html

```diff
@@ -41,22 +41,21 @@
 >
 
     <p>
-        You are receiving this email because one or more of your workloads in the Developer Sandbox for
-        Red Hat OpenShift has been running for 12 hours.
+        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for 12 hours.
```

Comment:
Not sure if others would agree but I think it'd be better to mention that it's been automatically idled as part of this paragraph to make it more clear that their workload has already been automatically idled.

```suggestion
        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for 12 hours and has been automatically idled.
```